pkg/tfbridge/main.go

```diff
@@ -69,20 +67,7 @@ func Main(pkg string, version string, prov ProviderInfo, pulumiSchema []byte, op
 	// Initialize Terraform logging.
 	prov.P.InitLogging()
 
-	if err := Serve(pkg, version, prov, pulumiSchema, opts...); err != nil {
+	if err := Serve(pkg, version, prov, pulumiSchema); err != nil {
 		cmdutil.ExitError(err.Error())
 	}
 }
-
-type Option func(*opts)
-
-type opts struct {
-	muxWith []func(*provider.HostClient) (pulumirpc.ResourceProviderServer, error)
-}
-
-// This is an experimental API.
-func MuxWith(s func(*provider.HostClient) (pulumirpc.ResourceProviderServer, error)) Option {
```

Comment:
This is not used anywhere? Should we go through a //deprecated step first before removing completely?

Reply:
I can't [find any usage](https://github.com/search?q=tfbridge.MuxWith&type=code) and I'm not 100% sure that it worked in the first place (untested). The API is marked as experimental. I'm comfortable just removing it.